src/AppBundle/Entity/Post.php

```diff
@@ -133,13 +133,13 @@ public function setAuthorEmail($authorEmail)
     /**
      * Is the given User the author of this Post?
      *
-     * @param User $user
+     * @param User|string $user
```

Comment:
The original code generated these failing tests:

![failed_tests](https://cloud.githubusercontent.com/assets/73419/8804399/68978f60-2fcc-11e5-973b-41b1d918041f.png)

Should we just remove those two tests?